src/main/scala/format/pgn/parsingModel.scala

```diff
@@ -89,12 +89,17 @@ case class InitialPosition(
     comments: List[String]
 )
 
+// could be factored as `PgnNode` and used directly in `ParsedPgn` too
+// not done for the moment for backward compat
+// see `GameNode` in python-chess
+case class Variation(comments: List[String], sans: Sans)
```

Comment:
For now I've design the patch to be as backward compatible as possible, but as noted here, further refactoring could be done

Reply:
I agree, we should overhaul this `parsingModel` at some point.